### DISCUSSION TITLE
Discover robot bugs

### DISCUSSION THREAD

**Discussion by zjgzh001** [trickerer/Trinity-Bots#475] [Classes]
When players board a flying vehicle, the robot will constantly move up and down, unable to fly
![image](https://github.com/trickerer/Trinity-Bots/assets/133203282/fb66bf89-c758-43b6-896f-42196b946036)
![image](https://github.com/trickerer/Trinity-Bots/assets/133203282/5ea343a2-308d-46de-a92b-b808b4492a90)

**Comment by trickerer** [comment]
https://github.com/trickerer/Trinity-Bots/commit/6a0eaff2106b4a25de87c56541eb8169428aeb89
